docs/up-logs/downloads/v1.md

```diff
@@ -11,13 +11,68 @@
 <div id="value-display"></div>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-Total number of times a work has been downloaded, as determined by processing the Access Logs of the website. Multiple downloads of the same work that occur over a single session will be counted as one download. A Session is a group of visits made by the same user within a continuous time frame, and ends after 30 minutes of inactivity.
+Total number of times a work has been downloaded, as determined by processing the Access Logs. To count multiple downloads of the same file by the same user, there must be a gap of 30 minutes from the last counted download.
 </div>
+
+<strong>
+For Example:
+</strong>
+
+<div style="text-align: center;">
+    <table style="margin: 0 auto;">
+        <tr>
+          <th>Time of download</th>
+          <th>Minutes from last download</th>
+          <th>downloads counted</th>
```

Comment:
For consistency, Downloads should be capitalized

Reply:
`<th>Downloads counted</th>`